pkg/block/azure/main_test.go

```diff
@@ -0,0 +1,100 @@
+package azure_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/service"
+	"github.com/ory/dockertest/v3"
+	"github.com/txn2/txeh"
+)
+
+const (
+	azuriteContainerTimeoutSeconds = 10 * 60 // 10 min
+	containerName                  = "container1"
+	accountName                    = "account1"
+	accountKey                     = "key1"
+)
+
+var (
+	pool     *dockertest.Pool
+	blockURL string
+)
+
+func runAzurite(dockerPool *dockertest.Pool) (string, func()) {
+	ctx := context.Background()
+	resource, err := dockerPool.Run("mcr.microsoft.com/azure-storage/azurite", "latest", []string{
+		fmt.Sprintf("AZURITE_ACCOUNTS=%s:%s", accountName, accountKey),
+	})
+
+	accountHost := fmt.Sprintf("%s.blob.localhost", accountName)
+
+	hosts, err := txeh.NewHostsDefault()
+	if err != nil {
+		panic(err)
+	}
+	hosts.AddHost("127.0.0.1", accountHost)
+
+	// set cleanup
+	closer := func() {
+		hosts.RemoveHost(accountHost)
+		err := dockerPool.Purge(resource)
+		if err != nil {
+			panic("could not kill postgres containerName")
+		}
+	}
+
+	// expire, just to make sure
+	err = resource.Expire(azuriteContainerTimeoutSeconds)
```

Comment:
I think you can pass that option in the docker run command.

Reply:
Not in the Run command itself